closed/OpenJ9.gmk

```diff
@@ -228,19 +228,32 @@ J9JCL_SOURCES_DONEFILE := $(MAKESUPPORT_OUTPUTDIR)/j9jcl_sources.done
 recur_wildcard = $(foreach dir,$(wildcard $1/*),$(call recur_wildcard,$(dir),$2) $(filter $(subst *,%,$2),$(dir)))
 AllJclSource   = $(call recur_wildcard,$(OPENJ9_TOPDIR)/jcl/src,*.java)
 
+JPP_CLASSPATH := $(OUTPUT_ROOT)/vm/sourcetools/lib/jpp.jar
+JPP_BASE_DIR  := $(OPENJ9_TOPDIR)
+JPP_SRC_ROOT  := jcl
+JPP_DEST      := $(SUPPORT_OUTPUTDIR)/j9jcl_sources
+
+ifneq (,$(CYGPATH))
+  # convert Cygwin paths to Windows paths with regular slashes
+  override JPP_BASE_DIR   := $(shell $(CYGPATH) -m $(JPP_BASE_DIR))
+  override JPP_SRC_ROOT   := $(shell $(CYGPATH) -m $(JPP_SRC_ROOT))
+  override JPP_DEST       := $(shell $(CYGPATH) -m $(JPP_DEST))
+  override JPP_CLASSPATH  := $(shell $(CYGPATH) -mp $(JPP_CLASSPATH))
+endif
```

Comment:
We should use `FixPath` from MakeBase.gmk like this so overrides are not necessary:
```
JPP_BASE_DIR := $(call FixPath,$(OPENJ9_TOPDIR))
JPP_JAR      := $(call FixPath,$(OUTPUT_ROOT)/vm/sourcetools/lib/jpp.jar)
JPP_DEST     := $(call FixPath,$(SUPPORT_OUTPUTDIR)/j9jcl_sources)
```
We don't need `JPP_SRC_ROOT`, and using JPP_JAR instead of JPP_CLASSPATH avoids the 'path' issue.
Please put these definitions in sorted order.

Reply:
Alternatively, we could just prefix the java JPP command line with `$(FIXPATH)`.

Reply:
I included the MakeBase.gmk  in OpenJ9.mk and fixed as suggested.